Doppler.HtmlEditorApi/Infrastructure/IRepository.cs

```diff
@@ -0,0 +1,12 @@
+using Doppler.HtmlEditorApi.Model;
+using System.Threading.Tasks;
+
+namespace Doppler.HtmlEditorApi.Infrastructure
+{
+    public interface IRepository
+    {
+        Task<ContentModel> GetCampaignModel(string accountName, int campaignId);
+        Task<TemplateModel> GetTemplateModel(string accountName, int templateId);
+        Task<TemplateModel> GetSharedTemplateModel(int templateId);
```

Comment:
Arguable, but in the convention is to use the `Async` suffix.

```suggestion
        Task<ContentModel> GetCampaignModelAsync(string accountName, int campaignId);
        Task<TemplateModel> GetTemplateModelAsync(string accountName, int templateId);
        Task<TemplateModel> GetSharedTemplateModelAsync(int templateId);
```

Reply:
Discussed and dismissed.